config.json

```diff
@@ -951,7 +951,16 @@
       "core": false,
       "difficulty": 8,
       "slug": "list-ops",
-      "topics": null,
+      "topics": [
+        "variables",
+        "lists",
+        "loops",
+        "filtering",
+        "searching",
+        "conditionals",
+        "callbacks",
+        "functional_programming"
```

Comment:
These look like good suggestions :) I agree with `functional_programming`, `filtering` and `lists`. `variables`, `loops` and `conditionals` are definitely used in this exercise but I wouldn't say they were important topics so I'm not sure they should be included. I'm not sure how `callbacks` and `searching` are relevant to the exercise. What do you think? :)

Reply:
And I would add `generics` as a topic :)

Reply:
I feel `variables` is too broad a term to apply honestly and I think `conditionals` don't come too much in play here either in my opinion. `loops` seems a decent shout so I think they might be okay to be included. I will update it accordingly.